vendor/vendor.json

```diff
@@ -226,12 +226,30 @@
 			"revision": "2ee87856327ba09384cabd113bc6b5d174e9ec0f",
 			"revisionTime": "2017-07-27T06:48:18Z"
 		},
+		{
+			"checksumSHA1": "jZpG8JA8KXXPB0ITK6ZWq1HUJNk=",
+			"path": "github.com/blinohod/terraform/builtin/providers/newrelic",
```

Comment:
Something is weird here, not sure why this is in the vendor on the PR

Reply:
I'm not sure on why that one is there, I'll remove it